ReactWindows/ReactNative.Net46/Views/View/ReactViewManager.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Data;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using ReactNative.Views.View;
+
+namespace ReactNative.Net46.Views.View
+{
```

Comment:
Fix namespace for consistency (remove `.Net46`)

Reply:
Fixed